js/client.js

```diff
@@ -2320,6 +2321,7 @@
 			buf += '<p><label class="optlabel"><input type="checkbox" name="nopastgens"'+(Tools.prefs('nopastgens')?' checked':'')+' /> Use modern sprites for past generations</label></p>';
 			buf += '<p><label class="optlabel"><input type="checkbox" name="notournaments"'+(Tools.prefs('notournaments')?' checked':'')+' /> Ignore tournaments</label></p>';
 			buf += '<p><label class="optlabel"><input type="checkbox" name="nolobbypm"'+(Tools.prefs('nolobbypm')?' checked':'')+' /> Don\'t show PMs in lobby chat</label></p>';
+			buf += '<p><label class="optlabel"><input type="checkbox" name="nounlinkhide"'+(Tools.prefs('nounlinkhide')?' checked':'')+' /> Don\'t hide locked/banned users\' chat messages</label></p>';
```

Comment:
I don't think users should have to click "show" to reveal messages if they don't want any hidden in the first place,